app/main/authorize/ayr_user.py

```diff
@@ -0,0 +1,68 @@
+from typing import List
+
+from sqlalchemy import exc
+
+from app.main.authorize.keycloak_manager import (
+    get_user_transferring_body_keycloak_groups,
+)
+from app.main.db.models import Body, db
+
+
+class AYRUser:
+    def __init__(self, groups: List[str]) -> None:
+        self.groups = groups
+
+    @property
+    def can_access_ayr(self):
+        return self.is_superuser or self.is_standard_user
+
+    @property
+    def is_superuser(self) -> bool:
+        return "/ayr_user_type/view_all" in self.groups
```

Comment:
Do we have the user types defined anywhere? Open to discussing this a bit more but might be good to define all the available user types if we know what the possible options are

Reply:
https://national-archives.atlassian.net/wiki/spaces/DAYR/pages/139100171/ADR-009+Keycloak+user+group+structure